src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetStatisticsRDD.scala

```diff
@@ -119,7 +120,7 @@ class ParquetStatisticsRDD(
 
   override def compute(split: Partition, context: TaskContext): Iterator[ParquetFileStatus] = {
     val configuration = hadoopConfiguration
-    val fs = FileSystem.get(configuration)
+    val fs = ParquetStatisticsRDD.getFS(configuration) 
```

Comment:
I'd prefer `getFileSystem` method name, but it does not matter that much, so it is up to you - feel free to change or keep the current name.